gui/default/index.html

```diff
@@ -393,6 +393,12 @@ <h4 class="panel-title">
                         <span>{{folderStats[folder.id].lastScan | date:'yyyy-MM-dd HH:mm:ss'}}</span>
                       </td>
                     </tr>
+                    <tr ng-if="folderStats[folder.id].nextScanDelta > 0">
+                      <th><span class="fa fa-fw fa-clock-o"></span>&nbsp;<span translate>Next Scan</span></th>
+                      <td class="text-right">
+                        <span>{{folderStats[folder.id].nextScan | date:'yyyy-MM-dd HH:mm:ss'}}</span>
+                      </td>
+                    </tr>
```

Comment:
Can we just have next time as a tooltip for the last scan? UI space is precious.